README.md

```diff
@@ -1,6 +1,18 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Calculator
+
+> This a calculator that solves advanced problems in Physics, Mathematics.
```

Comment:
It should be: This `is a` calculator that solves advanced problems in Physics, Mathematics.